documentation/release_notes.markdown

```diff
@@ -1,9 +1,31 @@
 ---
-title: "PuppetDB 2.2 » Release Notes"
+title: "PuppetDB 2.3 » Release Notes"
 layout: default
 canonical: "/puppetdb/latest/release_notes.html"
 ---
 
+2.3.0
```

Comment:
next release for master is '3.0.0' fwiw. But this could also be just 'master branch' or some place holder that could change later. We don't always know what version things are going to be, but this time we definitely are cutting a major, so its cool to leave this as 3.0.0 now.